src/tls.c

```diff
@@ -944,7 +951,7 @@ static int connTLSWritev(connection *conn_, const struct iovec *iov, int iovcnt)
      * and send it away by one call to connTLSWrite(). */
```

Comment:
@xbasel @ranshid is it ok? I'm not very happy of this but I don't have better idea, as this code now is handling 2 cases
1. combines a much bytes as possible in since syscall
2. provide enough buffer to retry last failed write
```suggestion
     * and send it away by one call to connTLSWrite().
     * IMPORTATNT: Code can fall back here in case when last write failed and first
     * element of iov is not big enough to provide required amount of bytes to retry,
     * so iov_bytes_len may exceed NET_MAX_WRITES_PER_EVENT by the amount
     * of remaining bytes from last taken iov. */
```